packages/@aws-cdk/aws-elasticsearch/test/integ.elasticsearch.ts

```diff
@@ -0,0 +1,29 @@
+import { EbsDeviceVolumeType } from '@aws-cdk/aws-ec2';
+import { App, Construct, Stack, StackProps } from '@aws-cdk/core';
+import * as es from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    new es.Domain(this, 'Domain', {
+      version: es.ElasticsearchVersion.V7_1,
+      ebs: {
+        volumeSize: 10,
+        volumeType: EbsDeviceVolumeType.GENERAL_PURPOSE_SSD,
+      },
+      logging: {
+        slowSearchLogEnabled: true,
+        appLogEnabled: true,
+      },
+      nodeToNodeEncryption: true,
+      encryptionAtRest: {
+        enabled: true,
+      },
+    });
+  }
```

Comment:
I think we should also add deployment of a basic auth cluster, as it poses several code path deviations.

Reply:
Added